Assets/Scripts/Control&Input/Peer2PeerTransport.cs

```diff
@@ -318,6 +379,10 @@ private void OnSpawnPlayerInput(NetworkConnectionToClient connection, PlayerConn
             connection.Send(new InitialPlayerDetailsMessage(existingPlayer));
         }
 
+        // Pick among the available colors
+        if (!availableColors.TryPop(out var color))
+            color = Color.white;
```

Comment:
Maybe just reset the color stack the same way as in the ResetState() instead of having white as backup?
Duplicate colors are a better backup imo